Dockerfile.dev

```diff
@@ -5,15 +5,23 @@ WORKDIR /app
 RUN npm install -g pnpm@7.11.0
 
```

Comment:
why Dockerfile.dev? It looks like it runs actually production? For dev its most likely to not use Docker at all

Reply:
my bad, forgot to delete that file